ui/css/logviewer.css

```diff
@@ -77,10 +77,29 @@ body {
     white-space: pre;
 }
 
+.lv-line-num {
+    white-space: pre;
+    float:left;
+    align-items: center;
+    border-right-style: solid;
+    border-right-color:  #F8EEC7;
+    border-right-width: thin;
+    -moz-user-select: none;
+    -khtml-user-select:none;
+    -webkit-user-select:none;
+    -o-user-select :none;
+    background-color: white;
```

Comment:
Actually let's remove this background property, looking at it locally I think having the alternate grey/white lines connected all the way through to the line number helps readability and relation to the line. We could always add it back in before merge if users feel differently.

I totally get why you did this originally though, to make it consistent with Github, thank you for doing that as a start point so we could evaluate it.